### DISCUSSION TITLE
錯誤率太高啦！Stack Overflow 禁止分享 ChatGPT 產生的答案

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#656] [General]
之前看到有同學介紹過ChatGPT，因此我就不多敘述了

知名程式設計問答網站 Stack Overflow[ 官方宣佈](https://meta.stackoverflow.com/questions/421831/temporary-policy-chatgpt-is-banned)，暫時禁止使用者分享由聊天機器人 ChatGPT 產生的回覆，根據官方表示：「由於從 ChatGPT 獲得正確答案的平均比例太低，發表由 ChatGPT 創造出的答案對網站上，不利於詢問或尋找正確答案的使用者。」
因為官方觀察到，因為  ChatGPT 快速方便，有許多人嘗試以此工具生成答案，但是在沒有專業知識或是無法在發表前驗證其正確性，在這些 AI 生成內容大量湧入之下，會對於基於志願者的管理平台產生影響。

我認為AI確實經過大量資料的學習，有時候真的會比人類聰明比人類懂得多，然而Stack Overflow的擔憂是否合理，也是值得我們思考與討論的，不知道各位有什麼看法呢？

來自https://www.inside.com.tw/article/29989-stack-overflow-banned-Chatgpt-ai-generated-answers

**Comment by 710388** [comment]
抓住了問題的關鍵，其他一切則會迎刃而解

**Comment by HsuChen88** [comment]
若有問題向ChatGPT提問，它說的都頭頭是道，但說了這麼多，實際上也不一定有幫助。

**Comment by Bonus502530** [comment]
我的室友曾經把線性代數的題目丟上ChatGPT上詢問，ChatGPT給了他一長串的計算過程，但是他仔細去看從頭到尾雖然寫的非常合理，但是最後根本解不出答案來，雖然ChatGPT有給出答案但是可以很明顯知道它給的答案是錯的，因此我認為Stack Overflow這樣的做法是正確的。

**Reply by liangyinglly** [reply]
哈哈哈 謝謝你有趣的分享

**Comment by 0212Jerry** [comment]
我認同stack overflow 的做法，畢竟錯誤的訊息或知識若氾濫的話，容易混淆他人。我們一開始使用Ai 的目的是希望他能在知識方面對人類有所幫助，錯誤的結果反而造成反效果

**Comment by Vincent20030314** [comment]
給了不確定或著不明確的答案等於沒給，甚至可能造成反效果。
